css/styles.css

```
@@ -43,200 +43,218 @@
 * {
     box-sizing: border-box;
     margin: 0;
-    
 }
-/*
- * Section one.  
- */
+
+body {
+    font-size: 9px;
+}
+
+/* Section I */
+
 .sectionI {
-   font-family: Roboto;  
+   font-family: Roboto;
 }
 
 .first_container {
     display: flex;
-    height: 496px;
-    padding-top: 15px;
+    padding:36px;
```

agrega un espacio después de los dos puntos.
```suggestion
    padding: 36px;
```